terraso_backend/apps/graphql/schema/projects.py

```diff
@@ -0,0 +1,80 @@
+# Copyright © 2021-2023 Technology Matters
```

Comment:
Suggestion: I think we need a new subfolder under `/graphql/schema` `site` and `project` are not self explainatory without knowing that is related to soild ID

Reply:
@josebui is this still relevant?

Reply:
I think we are good for now, I just think we should be careful about this, what the naming of this entities is saying is that they are not always related to Soil ID, we can have projects separated from soild ID for othe objectives, I think we should have that in mind when creating the other entities, and implementing other processes